### ISSUE TITLE
Show ranges as a normal user

### ISSUE THREAD

**Issue by mxwmwx** [cockroachdb/cockroach#48312]
![image](https://user-images.githubusercontent.com/37258498/80857328-fb1fef80-8c83-11ea-8c7a-b8c9d002c924.png)
As is shown,  the user must have the SELECT privilege on the target table and then he/she can use show ranges synopsis.However in cockroach,it seems that only superuser(root) or a member of admin has the privillege  to use "Show ranges".
So is that ok?

**Comment by rohany** [comment]
Yes, it looks like this is caused by the `crdb_internal.ranges` table requiring the admin role.
